src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,13 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo
+        todo={todo}
+        user={todo.user}
```

Comment:
why do you pass user property?

Reply:
you're right, i'm passing it later on in TodoInfo